templates/taxi/car_list.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+
+{% block content %}
+  {% if car_list %}
+    <h2>Manufacturer list</h2>
+    <table border="1px solid black">
+      <tr>
+        <th width="50px">id</th>
+        <th width="200px">Model</th>
```

Comment:
```suggestion
        <th style="width:200px" >Model</th>
```
Try this way everywhere you want to implement styles

Reply:
https://github.com/mate-academy/py-taxi-service-class-based-views/pull/191#issuecomment-1421128013